readme.md

```diff
@@ -14,33 +12,34 @@ We employ Terraform as the central tool for infrastructure-as-code, managing eve
 
 ### Ingress Configuration
 
-All services are exposed through an Nginx ingress that is presumed to already exist in the target cluster. This ingress handles routing and SSL termination, providing a unified access point to various services. Currently, self-signed certs are used due to the difficulty of using Let's Encrypt without a public endpoint. While this would be possible using DNS validation, this is complex to setup and requires a supported DNS provider.
+All services are exposed through an Nginx ingress that is presumed to already exist in the target cluster. This ingress handles routing and SSL termination, providing a unified access point to various services. Since the ingress of our cluster is not publicly available we route the traffic over the ingress of a different cluster. This second ingress controller handles SSL certificates via Let's Encrypt and is publicly available. 
 
-### it-rex.ch
+### meitrex.de
 
-We own the domain `it-rex.ch`, which is currently only used for exposing the Minio service. This is a workaround to the cluster's limitation of having a single DNS entry, `orange.informatik.uni-stuttgart.de`. Minio requires its own subdomain, making this arrangement necessary. The domain is owned and managed by Valentin (GitHub: v-morlock). It could also be used to expose the whole system under a nicer domain and possibly enable obtaining trusted SSL certs.
+We own the domain `meitrex.de`, which is currently only used for exposing the development version of MEITREX under the sub-domain `dev.meitrex.de`. The MINIO service endpoint under `minio.meitrex.de` and the corresponding dashboard under `minio-dashboard.meitrex.de`.
 
 ## Repository Structure
 
 ### General Infrastructure Resources
 
-- **main.tf**: Defines the Kubernetes namespace `gits` and establishes image pull secrets required for pulling Docker images from external repositories.
+- **main.tf**: Defines the Kubernetes namespace `meitrex`.
 - **ingress.tf**: Sets up the Nginx ingress for managing external access. Configurations for SSL redirection and proxy buffer sizes are also defined here. All services to be exposed have to be configured here.
 - **dapr.tf**: Deploys the Dapr runtime using Helm charts. Also includes the setup for state and pub-sub components using Redis.
 - **keel.tf**: Manages the deployment of Keel, a tool used for automated Kubernetes deployments, via Helm charts.
 - **keycloak.tf**: Handles the setup for Keycloak, used for identity and access management. It utilizes Helm charts for deployment and includes admin user and password settings.
+- **prometheus.tf**: Deploys the prometheus stack into a separate prometheus namespace to monitor the MEITREX application.
```

Comment:
It would be great to point to their documentation to know what the Prometheus stack contains. Is it only Prometheus and Grafana, or more?